src/utils/SiriusAdapter.cpp

```diff
@@ -420,14 +466,8 @@ class TOPPSiriusAdapter :
       QDir sw_dir(sirius_workspace_directory.toQString());
       sirius_workspace_directory = String(sw_dir.absolutePath());
       
-      // try to create directory if not present
-      if (!sw_dir.exists())
-      {
-        sw_dir.mkpath(sirius_workspace_directory.toQString());
-      }
-      
       // move tmp folder to new location
-      std::rename(tmp_dir.toStdString().c_str(), sirius_workspace_directory.c_str());
+      copyDirectoryFiles(tmp_dir, sirius_workspace_directory.toQString(), true);
```

Comment:
Handle false

Reply:
Please have a look at the new Version? 
Could you specify "handle false"?